api/src/pcapi/routes/backoffice/templates/accounts/get/general.html

```diff
@@ -148,6 +148,11 @@ <h6 class="card-subtitle text-muted">
             </div>
             {% if suspension_form %}
               {% if user.isActive %}
+                  {% if has_permission("MANAGE_PUBLIC_ACCOUNT") %}
+                  <div>
+                    {{ build_modal_form("unvalidate-account-password", password_invalidation_dst, password_invalidation_form, "Invalider le mot de passe", "Invalider le mot de passe", "Confirmer l'invalidation") }}
```

Comment:
```suggestion
                    {{ build_modal_form("invalidate-account-password", password_invalidation_dst, password_invalidation_form, "Invalider le mot de passe", "Invalider le mot de passe", "Confirmer l'invalidation") }}
```

Probablement a répercuter ailleurs